MarkBind/markbind
# Remove container fluid class from #app and add page content wrapper for styling

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x] Bug fix


Fixes #390



**What is the rationale for this request?**
User would like to standardise styling the site without having to deal with negative margins on the footer element.

**What changes did you make? (Give an overview)**
-  Localised `container-fluid` class to only components that specify it.
- Adjusted values in `site-nav.css` to compensate for removed padding around border.
- Introduce a unique `#content-wrapper` div for styling page contents. 

**Is there anything you'd like reviewers to focus on?**
- Is there a need to document overriding the `container-fluid` class with the user's custom CSS?


**Testing instructions:**
- Use `Netlify Preview` and remove the `website-content` class from inside the `page-content` div.
- Everything should look the same, in both desktop and responsive mode.
